.github/workflows/xcodebuild-or-fastlane.yml

```diff
@@ -86,11 +86,6 @@ on:
         required: false
         type: boolean
         default: false
-      cacheDerivedData:
```

Comment:
I think that removing a flag would be a breaking change; it might make sense to do a similar dedication warning using a `run` step as we do for the other deprecated elements.

Technically these workflows are public and might be referenced from outside the org so we should treat them as semantically versioned.

Reply:
I can implement a similar removal with depreciation warning to at least not break execution. 👍 
The feature will be nonetheless gone.

Reply:
Thank you 👍